azure-pipelines.yml

```diff
@@ -1,6 +1,10 @@
 variables:
 - name: _TeamName
   value: UnitTesting
+- name: _PublishUsingPipelines
+  value: true
+- name: _DotNetArtifactsCategory
```

Comment:
Not all artifacts built here are for .Net Core.
Just want to access the impact here. There are some artifacts, like the vsix and vsman which get inserted into the devdiv directly.

Reply:
I'm checking this with @singhsarab offline